src/ipz_parser.cpp

```diff
@@ -631,4 +631,62 @@ types::DbusVariantType IpzVpdParser::readKeywordFromHardware(
     return types::DbusVariantType{
         getKeywordValueFromRecord(l_record, l_keyword, l_recordOffset)};
 }
+
+bool IpzVpdParser::isRecordECCValid(const auto& i_recordDataOffset,
```

Comment:
What is the use case of this API? Why is it needed, I could not understand from commit message.